.github/workflows/dapr.yml

```diff
@@ -51,6 +51,10 @@ jobs:
             target_os: linux
           - os: windows-2019
             target_os: windows
```

Comment:
Should this entry have a `windows_version` too?

Reply:
No @ItalyPaleAle, adding a `windows_version` here will start creating `daprio/daprd:1.10.2-windows-1809-amd64` images, which we don't want. The current 1809 image is `daprio/daprd:1.10.2-windows-amd64`, which remains unchanged. Going forward any Windows version will be of the format `daprio/daprd:<dapr-version>-windows-<windows-version>-<arch>`.